lib/CXGN/BrAPI/v2/ObservationVariables.pm

```diff
@@ -463,4 +519,72 @@ sub observation_variable_ontologies {
     return CXGN::BrAPI::JSONResponse->return_success(\%result, $pagination, \@data_files, $status, 'Ontologies result constructed');
 }
 
+sub _construct_variable_response {
+    my $self = shift;
+    my $c = shift;
+    my $variable = shift;
+
+    my $external_references_json;
+    if (defined($variable->external_references)) {
+        $external_references_json = $variable->external_references->search()->{$variable->cvterm_id};
+
+        # TODO figure out how to integrate this with external references stored in the db
+        # push @external_references_json, {
+        #     referenceID => "http://www.cropontology.org/terms/".$db_name.":".$accession . "/",
+        #     referenceSource => "Crop Ontology"
+        # };
```

Comment:
We will create a feature flag to determine if this xref should be added to a variable or not and include that in this PR